### ISSUE TITLE
StreamingMultipartFormDataParser FileHandler data providing

### ISSUE THREAD

**Issue by KreutzerCode** [Http-Multipart-Data-Parser/Http-Multipart-Data-Parser#86]
Question
Are the examples of the README.md regarding the StreamingMultipartFormDataParser for large streams with large files up to date? I have problems implementing these examples as the FileHandler seems to provide the data differently now.

**Comment by Jericho** [comment]
I updated the samples in the reader a few months ago so as far as I know, they are up-to-date. What is it that you are having problems with? Do you mind sharing your code so I can have a look?

**Comment by KreutzerCode** [comment]
Thanks for the quick response!
Yeah, sure, I created a method that handles large input streams for my lokal http server. There I want to use the StreamingMultipartFormDataParser. I am currently using version 4.3.1.

![GithubQuestion](https://user-images.githubusercontent.com/53813841/87548663-1a3bf400-c6ad-11ea-9b95-b47ada35c174.png)

The example which I follow I have taken completely from the Readme.md. 

What I was able to figure out is that when I downgrade the version of the HttpMultipartDataParser to version 3.1.0, the error no longer appears.
